sdk/monitor/azure-monitor-query/azure/monitor/query/_models.py

```diff
@@ -143,7 +153,7 @@ def _from_generated(cls, generated):
 class MetricsList(list):
     """Custom list for metrics
     """
-    def __init__(self, **kwargs): # pylint: disable=super-init-not-called
+    def __init__(self, **kwargs: Any): # pylint: disable=super-init-not-called
```

Comment:
nit: for type-completeness sake

```suggestion
    def __init__(self, **kwargs: Any) -> None: # pylint: disable=super-init-not-called
```